api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-list-account.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "List account"
+description: "Get the userAccountInformations from the account navigation property."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List account
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Get the userAccountInformations from the account navigation property.
```

Comment:
Rather than try and guess pluralization we should do the following, but only if this is valuable information, otherwise we should just put a TODO.  Angela/Laura - need your help on the best wording for a navigation property...  Here's my attempt, but this essentially is repeated in the response section...

List {nav_property_name} which returns a collection of {reference to the resource topic}.

ASIDE:  IMHO API review should have caught this - this nav property should be called `accounts`, not `account`.

Reply:
Re making the "account" nav property plural - if @kevinbellinger has a few updates to make to the profile API in beta, may not be a bad idea to correct this when the API is still in beta only.

Re the generated boilerplate description, how about adding "instances" at the end of the sentence?
"List {nav_property_name} which returns a collection of {reference to the resource topic} instances"

Reply:
@angelgolfer-ms I think we use "objects" rather than "instances" elsewhere
BUG BUG

Reply:
How about "List {nav_property_name} which returns a collection of {reference to the resource topic} resources"?

You're right we've got "objects" in a few places, not having editors doesn't help, and we should not perpetuate it. :-|

According to odata.org, I think we should use "resources":
https://www.odata.org/getting-started/understand-odata-in-6-steps/
RFC definition:
https://tools.ietf.org/html/rfc2616#section-1.3
Our own OData REST API guidance:
https://docs.microsoft.com/en-us/graph/use-the-api?view=graph-rest-beta
https://docs.microsoft.com/en-us/graph/traverse-the-graph?view=graph-rest-beta